src/sas/qtgui/Utilities/PreferencesPanel.py

```diff
@@ -0,0 +1,169 @@
+import logging
```

Comment:
Why does git think that this code is all entirely new? Think something might have gone wrong somewhere.

Reply:
probably while merging `main` back and manually resolving conflict?